cpp/ps0005/cpp_ps0005.cpp

```diff
@@ -37,12 +14,7 @@ int main()
       std::cout << "Enter the operator + - * / % :" << std::endl;
       std::cin >> op;

```

Comment:
- [x] declared a variable to store the user input
- [x] chose a suitable data type for the variable
- [x] prompted user to enter a number
- [x] used `std::cin` in a right way to get user's input